### DISCUSSION TITLE
SPORT TV Uzbekistan

### DISCUSSION THREAD

**Discussion by pukozs75** [iptv-org/community#784] [Q&A]
### Channel Name

SPORT UZ

### Channel ID

Sport.uz

### Website

https://spbtv.online/channels/sport-uz.html

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19752
